uc.c

```diff
@@ -369,6 +369,14 @@ uc_err uc_close(uc_engine *uc)
     return UC_ERR_OK;
 }
 
+UNICORN_EXPORT
+void uc_set_slow_self_unpack(uc_engine *uc, bool state)
```

Comment:
Could you provide some examples on how to use this API? It also helps us decide whether this API is essential.